web/router.go

```diff
@@ -38,10 +43,64 @@ func SetupRoutes(router *gin.Engine) {
 	router.Use(middlewares.ParseHost())
 	router.Use(middlewares.ServeApp(apps.Serve))
 	router.Use(middlewares.ErrorHandler())
+	router.Use(corsMiddleware("/apps", "/data", "/files"))
```

Comment:
OK, you should note this in a comment. It could be helpful later to remember that